### PR TITLE
feat: add sonarlint-language-server

### PR DESCRIPTION
**Pull request by huesersohn**

### CODE REVIEW

File: packages/sonarlint-language-server/package.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+source:
+  id: pkg:github/SonarSource/sonarlint-vscode@3.17.0
+  asset:
+    file: sonarlint-vscode-{{version}}.vsix
+
+bin:
+  sonarlint-language-server: java-jar:extension/server/sonarlint-ls.jar
+
+share:
+  sonarlint-analyzers: extension/analyzers
```
Comment (williamboman) [inline packages/sonarlint-language-server/package.yaml:32]:
```suggestion
  sonarlint-analyzers: extension/analyzers/
```

**Review by williamboman — APPROVED** [review]
Thanks! I think it might make sense to add support for a `mvn` package type as jars are published on Maven. This asset file contains a bunch of additional, unused, files of ~100 MiB

**Comment by huesersohn** [comment]
The bulk of the size (124 MB) comes from the analyzers that I link to shared as they are needed for the various filetypes. Apart from that the Server is 23 MB, so that leaves about 3 or 4 MB for the rest of the files.

**Comment by williamboman** [comment]
> The bulk of the size (124 MB) comes from the analyzers that I link to shared as they are needed for the various filetypes. Apart from that the Server is 23 MB, so that leaves about 3 or 4 MB for the rest of the files.

Ah, I see 👍

**Review by williamboman — APPROVED** [review]
Thanks again, on mobile so had to rely on CI for the needed adjustments. Should pass now 🤞

**Comment by huesersohn** [comment]
Thank you!

**Pull request merged**